experiments/ssl4eo/download_l5_raw.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
```

Comment:
this isn't working on my windows machine??

Reply:
What error are you getting?

Reply:
oh I'm joking, its a bash script